user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -35,7 +35,8 @@ encryption                       | no        |
 access logging                   | no        |
 website hosting configuration    | no        |
 
-Setting bucket/object ACLs is only partially possible and currently unintuitive to use. We are in contact with the software manufacturer to solve this problem. For detailed information feel free to contact our [Cloud Support](../../06.Support/default.en.md).
+For setting up bucket/object ACLs we suggest to use [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html). We have prepared a [guide in our Tutorials section](../../02.Tutorials/11.object-storage-acls/docs.en.md).
```

Comment:
This change removes the explanation of "partial", but leaves the word "partial" in the table above. As I don't feel like taking the responsability for full ACL support, so we should not necessarily change it to a "yes", but then keep an explanation (although different) for what we mean by "partial". For example, it seems that access to the index cannot be restricted by ACL, as in AWS and even if this unexpectedly works after latest updates, there will likely remain differences.

Reply:
Good point, lets have another closer look what AWS offers for features. I am not sure if the directory listing part is more a topic of the bucket policies (see the ACL part of their docs : https://docs.aws.amazon.com/AmazonS3/latest/dev-retired/acl-overview.html). It may even be achieved by narrowing down ACLs to WRITE only (depending if that is practicable for the customer use-case) 

We should refer to the canned ACLs as the implementation as not all of these predefined ACLs even exists in our implementation.

Reply:
Added a further note to make clear we do not offer the complete Amazon S3 feature set (I wish we would tho).

Reply:
Issue handled in another comment! =)